app/controllers/patients_controller.rb

```diff
@@ -314,6 +314,18 @@ def update_status
     patient = current_user.get_patient(params.permit(:id)[:id])
     redirect_to(root_url) && return if patient.nil?
 
+    # update LDE for patient and group members only in the exposure workflow with continuous exposure on (separate from updating monitoring status)
+    if params.permit(:apply_to_group_cm_exp_only)[:apply_to_group_cm_exp_only] && params[:apply_to_group_cm_exp_only_date].present?
+      (current_user.get_patient(patient.responder_id)&.dependents&.where(continuous_exposure: true, isolation: false) || []).uniq.each do |member|
```

Comment:
You'll want to do a `dependents_exclude_self` here I believe, since this is not supposed to include the HoH and dependents includes all dependents + itself for reporters.

Reply:
good catch - pushed